### PR TITLE
Fix #123 - Update to Placeholder Images topic

### LINKED ISSUE #123: Feedback on page: /catalog/product-image-placeholders.html
It would be good to have the images sizes to upload on this page if possible

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) adds a reference link to image roles and sizing in the Placeholder topic. Formatting updates also implemented.

## Affected documentation pages

https://docs.magento.com/m2/ee/user_guide/catalog/product-image-placeholders.html

## Affected Magento editions

- [X] Open Source
- [X] Commerce
- [X] B2B

### CODE REVIEW

File: src/catalog/product-image-placeholders.md [hunk 1]
```diff
@@ -5,19 +5,21 @@ title: Placeholders
 Magento uses temporary images as placeholders until the permanent product images become available. A different placeholder can be uploaded for each role. The initial placeholder image is the Magento logo, which you can replace with an image of your choice.
 
 ![]({% link images/images/storefront-image-placeholder.png %}){: .zoom}
-*Image Placeholder*
+_Image Placeholder_
 
-## To upload placeholder images:
+## Upload Placeholder Images:
```
Comment (leslietilling) [inline src/catalog/product-image-placeholders.md]: remove the colon (:) - linting rule will flag that

File: src/catalog/product-image-placeholders.md [hunk 2]
```diff
@@ -5,19 +5,21 @@ title: Placeholders
...
 
-1. On the Admin sidebar, tap **Stores**. Then under **Settings**, choose **Configuration**.
+1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
 
-1. In the panel on the left under **Catalog**, and choose **Catalog**.
+1. In the panel on the left under **Catalog**, choose **Catalog**.
```
Comment (leslietilling) [inline src/catalog/product-image-placeholders.md]:
```suggestion
1. In the left panel, expand **Catalog** and choose **Catalog** underneath.
```

File: src/catalog/product-image-placeholders.md [hunk 3]
```diff
@@ -5,19 +5,21 @@ title: Placeholders
...
 
-1. Expand ![]({% link images/images/btn-expand.png %}){: .Inline} the **Product Image Placeholders** section.
+1. Expand ![Expansion icon]({% link images/images/btn-expand.png %}){: .Inline} the **Product Image Placeholders** section.
 
-1. For each image role, tap <span class="btn"> Choose File </span>. Find the image on your computer and upload the file. You can use the same image for all three roles, or upload a different placeholder image for each role.
+1. For each image role, click <span class="btn"> Choose File </span>. Find the image on your computer and upload the file. You can use the same image for all three roles, or you can upload a different placeholder image for each role.
```
Comment (leslietilling) [inline src/catalog/product-image-placeholders.md]: remove spaces in button span tag

File: src/catalog/product-image-placeholders.md [hunk 4]
```diff
@@ -5,19 +5,21 @@ title: Placeholders
...
 
     ![]({% link images/images/config-catalog-catalog-product-image-placeholders.png %}){: .zoom}  
-    *[Product Image Placeholders]({% link configuration/catalog/catalog.md %})*
+    _[Product Image Placeholders]({% link configuration/catalog/catalog.md %})_
 
-1. When complete, tap <span class="btn"> Save </span>.
+1. When complete, click <span class="btn"> Save </span>.
```
Comment (leslietilling) [inline src/catalog/product-image-placeholders.md]: remove spaces in button span tag

File: src/catalog/product-image-placeholders.md (hunk 3 above)
Comment (hickskenh) [reply src/catalog/product-image-placeholders.md]: done

File: src/catalog/product-image-placeholders.md (hunk 4 above)
Comment (hickskenh) [reply src/catalog/product-image-placeholders.md]: removed in 2 places

**Comment by hickskenh** [comment]
running tests

**Pull request merged**